.circleci/config.yml

```
@@ -228,12 +234,10 @@ workflows:
               path: tmp/test-results
         matrix:
           parameters:
-            ruby-version:
-            - '3.0'
-            - '3.1'
-            - '3.2'
-            - '3.3'
-            gem-version:
-            - rails-6.1.7
-            - rails-7.0.8
-            - rails-7.1.3
+            ruby-version: ['3.0', '3.1', '3.2', '3.3']
+            gem-version: ['rails-7.0.8', 'rails-7.1.3', 'rails-8.0.2']
+          exclude:
+            - ruby-version: '3.0'
+              gem-version: 'rails-8.0.2'
+            - ruby-version: '3.1'
+              gem-version: 'rails-8.0.2'
```

```suggestion
            ruby-version: ['3.2', '3.3']
            gem-version: ['rails-7.0', 'rails-7.1', 'rails-8.0']
```

Similar to https://github.com/doximity/super_spreader/pull/76#discussion_r2298854106

Better to just not run the EOL ruby versions at all in my view

---

Adding ruby 3.4 is another option but you _might_ do that in a separate PR, that does tend to come with some gem dependency adjustments since 3.4 discontinued several default gems (eg `bigdecimal`, `mutex_m`) https://stdgems.org/new-in/3.4/